src/main/test/com/epam/preprod/sirenko/FourElementsOfStringSumAsHashCodeWrapperTest.java

```diff
@@ -0,0 +1,108 @@
+package com.epam.preprod.sirenko;
+
+import com.epam.preprod.sirenko.entity.DryFood;
+import com.epam.preprod.sirenko.entity.Food;
+import com.epam.preprod.sirenko.entity.Product;
+import com.epam.preprod.sirenko.enums.PetGroup;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class FourElementsOfStringSumAsHashCodeWrapperTest {
+	
+	private static Food food;
+	private static Food otherFood;
+	private static DryFood dryFood;
+	private static DryFood otherDryFood;
+	private static Map<FourElementsOfStringSumAsHashCodeWrapper, Product> hashMap;
+	private static Map<FourElementsOfStringSumAsHashCodeWrapper, Product> linkedHashMap;
+	private static FourElementsOfStringSumAsHashCodeWrapper abracadabra;
+	private static FourElementsOfStringSumAsHashCodeWrapper blabla;
+	private static FourElementsOfStringSumAsHashCodeWrapper anotherBlabla;
+	private static FourElementsOfStringSumAsHashCodeWrapper moonlight;
+	private static FourElementsOfStringSumAsHashCodeWrapper moon;
+	private static FourElementsOfStringSumAsHashCodeWrapper hooray;
+	
+	
+	@BeforeAll
+	static void setData() {
+		hashMap = new HashMap<>();
+		linkedHashMap = new LinkedHashMap<>();
+		abracadabra = new FourElementsOfStringSumAsHashCodeWrapper("abracadabra");
```

Comment:
We can create private method setupWrappers that we can call here,and move setup for wrappers to it

Reply:
done